SIT/smoke/01_basic_kvp/cdm.sh

```diff
@@ -21,4 +21,4 @@ EOF
 spark-submit \
   --properties-file /smoke/01_basic_kvp/migrate.properties \
   --master "local[*]" \
-  --class datastax.astra.migrate.DiffData /local/cassandra-data-migrator.jar
+  --class datastax.cdm.job.DiffData /local/cassandra-data-migrator.jar
```

Comment:
shouldn't this be `com.datastax.cdm.job.DiffData` as the class is [here](https://github.com/datastax/cassandra-data-migrator/blob/main/src/main/scala/com/datastax/cdm/job/DiffData.scala)?

Reply:
It should be --class com.datastax.cdm.job cc: @mieslep , will raise a separate JIRA ticket to resolve the refactoring issues  synk security check.